lib/utils.pm

```diff
@@ -1077,4 +1078,20 @@ sub script_run_interactive {
     }
 }
 
+
+=head2 sed_config
+  sed_config("filename",{text_to_find => text_to_replace})
+
+  generify sed usage as config file modification tool.
+  allow to modify several items in one function call 
+  by providing  text_to_find / text_to_replace as hash key/value pairs
+=cut
+sub sed_config {
+	my($filename, $to_replace) = @_;
```

Comment:
What do you think about using a hash. So a call to that function would look like:

```
sed_config($filename, a => 'b', c => 'd');
```

```suggestion
	my($filename, %to_replace) = @_;
```